### ISSUE TITLE
[Feature] Focus Frame Anchor Option

### ISSUE THREAD

**Issue by lolswirl** [Krealle/Cell_UnitFrames#84]
### Describe the feature

Give focus frame an "anchor to X" frame with a choice of what it anchors to.

### Describe the implementation

I really just want to anchor my focus frame to my player frame

**Comment by Krealle** [comment]
I'll add the option to anchor it to the Player frame for now. Adding the option to choose which frame specifically it anchors to will be a future implementation.

**Comment by zandadoum** [comment]
> I'll add the option to anchor it to the Player frame for now. Adding the option to choose which frame specifically it anchors to will be a future implementation.

yeah i'd like to have this feature aswell for all the frames: instead of a button to "anchor to x" have a dropdown where we can select to what to anchor it. that would be fantastic :D
